code/modules/overmap/spacetravel.dm

```diff
@@ -1,26 +1,36 @@
 //list used to cache empty zlevels to avoid nedless map bloat
+var/man_overboard_counter = 1 // used for naming the man overboard sites in case multiple appear in close proximity
 var/list/cached_space = list()
 
 //Space stragglers go here
 
 /obj/effect/overmap/visitable/sector/temporary
-	name = "Deep Space"
-	invisibility = 101
-	known = 0
+	name = "Man Overboard"
+	known = FALSE
+	layer = OVERMAP_IMPORTANT_SECTOR_LAYER
 
 /obj/effect/overmap/visitable/sector/temporary/New(var/nx, var/ny, var/nz)
+	name = man_overboard_counter > 1 ? "[name] ([man_overboard_counter])" : name
 	loc = locate(nx, ny, current_map.overmap_z)
 	x = nx
 	y = ny
 	map_z += nz
 	map_sectors["[nz]"] = src
 	testing("Temporary sector at [x],[y] was created, corresponding zlevel is [nz].")
+	new /obj/effect/shuttle_landmark/automatic(locate(127, 127, nz))
```

Comment:
pretty sure using locate is slower than actually just setting the XYZ coordinates

Reply:
it sets its landmark tag on initialize, so doesn't it need to be spawned onto its loc during the new?